src/sdl2_wrapper.cpp

```diff
@@ -717,17 +717,39 @@ CAMLprim value Val_SDL_Event(SDL_Event *event) {
   case SDL_PANEVENT:
     v = caml_alloc(1, 24);
 
-    vInner = caml_alloc(9, 0);
+    vInner = caml_alloc(40, 0);
```

Comment:
5 is enough ? `panEvent` has 5 field: so 5 values.

Reply:
🤦 yep, forgot it isn't malloc lol

Reply:
Fixed